src/Components/Passage.tsx

```diff
@@ -31,8 +31,8 @@ interface PassageProps {
 const ChatContainer = styled.div<{ hasHeader?: boolean }>`
   display: flex;
   height: ${({ hasHeader }) => (hasHeader ? 'calc(100% - 4.5rem)' : '100%')};
-  overflow: scroll;
   flex-direction: column;
+  overflow: auto;
```

Comment:
Just because we don't intend to scroll in x-axis

```suggestion
  overflow-y: auto;
```

Reply:
was just about to write.
 we have the extra-buildings screen where if you have more buildings on a small screen we need to scroll left or right.

Reply:
Oh that's very annoying... should probably have another solution for that specifically but I will approve for now!

Reply:
nevermind. overflow-y seems to work as well. the extra-building already has scroll implemented when there are multiple boxes. 

tested on several flows and seems to work.